shared/src/iosMain/kotlin/co/touchlab/kampstarter/NativeCoroutineAdapter.kt

```diff
@@ -0,0 +1,53 @@
+package co.touchlab.kampstarter
+
+import co.touchlab.kampstarter.db.Breed
+import co.touchlab.kampstarter.models.BreedModel
+import co.touchlab.kampstarter.models.ItemDataSummary
+import co.touchlab.kermit.*
+import kotlinx.coroutines.*
+import kotlinx.coroutines.flow.*
+import co.touchlab.stately.ensureNeverFrozen
+
+
+class NativeCoroutineAdapter(
+    private val viewUpdate: (ItemDataSummary) -> Unit,
+    errorUpdate: (String) -> Unit
+) {
+
+    val log = Kermit(CommonLogger())
```

Comment:
Should this be getting a logger from Koin instead of creating a brand new one?

Reply:
I've changed it to get it from the base BaseModel, which is getting it via injection